massa-bootstrap/src/tests/tools.rs

```diff
@@ -269,6 +291,79 @@ pub fn assert_eq_bootstrap_graph(v1: &BootstrapableGraph, v2: &BootstrapableGrap
     }
 }
 
+pub fn get_execution_state() -> BootstrapExecutionState {
+    BootstrapExecutionState {
+        final_slot: Slot::new(14, 1),
+        final_ledger: SCELedger(
+            vec![
+                (
+                    get_random_address(),
+                    SCELedgerEntry {
+                        balance: Amount::from_str("129").unwrap(),
+                        opt_module: None,
+                        data: vec![
+                            (
+                                massa_hash::hash::Hash::from("key_testA".as_bytes()),
+                                "test1_data".into(),
+                            ),
+                            (
+                                massa_hash::hash::Hash::from("key_testB".as_bytes()),
+                                "test2_data".into(),
+                            ),
+                            (
+                                massa_hash::hash::Hash::from("key_testC".as_bytes()),
+                                "test3_data".into(),
+                            ),
+                        ]
+                        .into_iter()
+                        .collect(),
+                    },
+                ),
+                (
+                    get_random_address(),
+                    SCELedgerEntry {
+                        balance: Amount::from_str("878").unwrap(),
+                        opt_module: Some("bytecodebytecode".into()),
+                        data: vec![
+                            (
+                                massa_hash::hash::Hash::from("key_testD".as_bytes()),
+                                "test4_data".into(),
+                            ),
+                            (
+                                massa_hash::hash::Hash::from("key_testE".as_bytes()),
+                                "test5_data".into(),
+                            ),
+                        ]
+                        .into_iter()
+                        .collect(),
+                    },
+                ),
+            ]
+            .into_iter()
+            .collect(),
+        ),
+    }
+}
+
+pub fn assert_eq_exec(v1: &BootstrapExecutionState, v2: &BootstrapExecutionState) {
```

Comment:
Could not that be an impl of Eq for &BootstrapExecutionState ?

Reply:
It's quite test-specific